src/topology/continuous_function/algebra.lean

```diff
@@ -113,6 +113,28 @@ instance {α : Type*} {β : Type*} [topological_space α] [topological_space β]
   ..continuous_map.semigroup,
   ..continuous_map.has_one }
 
+@[to_additive]
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {β : Type*} [monoid β] [topological_space β]
+  [has_continuous_mul β] : has_continuous_mul C(α, β) :=
```

Comment:
```suggestion
instance {α : Type*} {β : Type*} [topological_space α] [locally_compact_space α]
  [monoid β] [topological_space β] [has_continuous_mul β] :
  has_continuous_mul C(α, β) :=
```
(seems to be the standard order in this section, see the previous lemma)

Reply:
Done